assets/javascripts/discourse/components/discourse-post-event/index.gjs

```diff
@@ -0,0 +1,198 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import willDestroy from "@ember/render-modifiers/modifiers/will-destroy";
+import { service } from "@ember/service";
+import { htmlSafe } from "@ember/template";
+import concatClass from "discourse/helpers/concat-class";
+import routeAction from "discourse/helpers/route-action";
+import { emojiUnescape } from "discourse/lib/text";
+import { escapeExpression } from "discourse/lib/utilities";
+import i18n from "discourse-common/helpers/i18n";
+import I18n from "discourse-i18n";
+import Creator from "./creator";
+import Dates from "./dates";
+import Invitees from "./invitees";
+import MoreMenu from "./more-menu";
+import Status from "./status";
+import Url from "./url";
+
+export default class DiscoursePostEvent extends Component {
+  @service currentUser;
+  @service discoursePostEventApi;
+  @service messageBus;
+
+  @action
+  setupMessageBus() {
+    this.messageBus.subscribe(
+      "/discourse-post-event/" + this.args.event.post.topic.id,
+      async (msg) => {
+        const event = await this.discoursePostEventApi.event(msg.id);
+        this.args.event.updateFromEvent(event);
+      }
+    );
+  }
+
+  @action
+  teardownMessageBus() {
+    this.messageBus.unsubscribe(
+      "/discourse-post-event/" + this.args.event.post.topic.id
+    );
+  }
+
+  get eventStatusLabel() {
+    return I18n.t(
+      `discourse_calendar.discourse_post_event.models.event.status.${this.args.event.status}.title`
+    );
+  }
+
+  get eventStatusDescription() {
+    return I18n.t(
+      `discourse_calendar.discourse_post_event.models.event.status.${this.args.event.status}.description`
+    );
```

Comment:
if we're using only `I18n.t()` and already importing the `i18n` helper, then we can use just the helper and save ourselves one import

```suggestion
    return i18n(
      `discourse_calendar.discourse_post_event.models.event.status.${this.args.event.status}.title`
    );
  }

  get eventStatusDescription() {
    return i18n(
      `discourse_calendar.discourse_post_event.models.event.status.${this.args.event.status}.description`
    );
```